src/app/components/toast/toast.module.scss

```
@@ -0,0 +1,39 @@
+@use "@styles/mixins" as mixin;
+@use "@styles/typography" as typo;
+@use "@styles/colors" as color;
+
+.toast-warpper {
```

재형님, 클래스명에 오타가 있는 것 같습니다! 😂 

```suggestion
.toast-wrapper {
```